### PR TITLE
MEN-5124: do not hard-code the email sender, rely on workflows' default

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
Combined QA pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/379260004

File: client/orchestrator/client_orchestrator_test.go [hunk 1]
```diff
@@ -259,12 +258,11 @@ func TestSubmitDeviceLimitWarning(t *testing.T) {
 		Error: errors.New(
 			`workflows: \[internal\] invalid request argument: ` +
 				`invalid device limit request: ` +
-				`missing parameter "from"`),
+				`missing parameter "to"`),
```
Comment (merlin-northern) [inline client/orchestrator/client_orchestrator_test.go:261]: wait I do not get it, now "to" is missing?

**Review by merlin-northern — APPROVED** [review]
lgtm with some nitpick in tests

File: client/orchestrator/client_orchestrator_test.go (hunk 1 above)
Comment (tranchitella) [reply client/orchestrator/client_orchestrator_test.go:261]: from is not mandatory anymore (that's the change)

**Pull request merged**